content/rc/security/single-sign-on/saml-sso/saml-integration-auth0.md

```diff
@@ -0,0 +1,194 @@
+---
+Title: Auth0 SAML integration guide
+linkTitle: Auth0 integration
+description: This integration guide shows how to configure Auth0 as a SAML single sign on provider for your Redis Cloud account.
+weight: 58
+alwaysopen: false
+categories: ["RC"]
+---
+
+This guide shows how to configure [Auth0](https://auth0.com/docs) as a SAML single sign on identity provider (IdP) for your Redis Cloud account.
+
+To learn more about Redis Cloud support for SAML, see [SAML single sign on]({{<relref "/rc/security/single-sign-on/saml-sso">}}).
+
+## Step 1 - Setup your identity provider (IdP)
+
+### Prepare the SM "owner" user
+
+1. Sign in in your Auth0 account and open **User Management → Users**
+   * Select the User that will be SAML owner in SM
+   * Verify the details
+
+    First & Last names are needed for the SAML assertion and are not available in the default user profile. 
+
+    > **Note**: Depending how the users are created, these can have different profiles, see: Sample User Profiles
+
+    {{<image filename="images/rc/saml/auth0_saml_1.png" alt="" >}}{{</image>}}
+
+2. Add **user_metadata** to fulfill the SAML assertion then click the **Save** button. The key-value pair of **redisAccountMapping** consists of the **lower-cased role name** (ie owner, member, manager, or viewer) AND your **Redis Cloud Account ID** (you can find this information in the upper-right user menu at app.redislabs.com).
```

Comment:
```suggestion
2. Add **user_metadata** to fulfill the SAML assertion then click the **Save** button. The key-value pair of **redisAccountMapping** consists of the **lower-cased role name** (owner, member, manager, or viewer) AND your **Redis Cloud Account ID** found in the [account settings]({{<relref "rc/accounts/account-settings">}}).
```

We usually don't use directional language such as upper-right, so adding a reference to where you can find the account ID.